routes/index.js

```diff
@@ -15,9 +15,26 @@ router.get('/', function(req, res) {
 
 /* GET /notebooks - list of notebooks */
 router.get('/notebooks', function(req, res) {
-    res.render('index', {
-        title: 'Notebooks'
-    });
+    nbstore.getNotebooks().then(
+        function success(notebooks) {
+            //if notebook index exists redirect to it immediately
+            var notebookIndex = "index.ipynb"
+            if(notebooks.indexOf(notebookIndex) > -1) {
+                res.redirect('/notebooks/' + notebookIndex);
+            }
+            else {
+                //render a list of all notebooks
+                res.render('index', {
+                    username: req.session.username,
+                    notebooks: notebooks
+                });
+            }
+        },
```

Comment:
If there is an "index" notebook, then we don't show the list. I think it would be good to have a "fallback" which always exists, such as the route `/list`.  If no "index", then `/` shows the same as `/list`. If there is an "index", it's shown at `/`, but `/list` is still accessible.  @parente, what do you think?